llvm/test/Transforms/Attributor/value-simplify-pointer-info.ll

```diff
@@ -231,13 +231,13 @@ entry:
   %add15 = add nsw i32 %i10, %i11
   %i316 = getelementptr inbounds %struct.S, ptr %agg.result, i64 0, i32 2
   store i32 %add15, ptr %i316, align 4, !tbaa !14
-  call void @llvm.lifetime.end.p0(ptr nonnull %s)
+  call void @llvm.lifetime.end.p0(i64 24, ptr nonnull %s)
   ret void
 }
 
-declare void @llvm.lifetime.start.p0(ptr nocapture)
+declare void @llvm.lifetime.start.p0(i64 immarg, ptr nocapture)
```

Comment:
It doesn't seem like this file change is related to this PR. Can you pre-commit these?

Reply:
does this mean create a separate PR for these changes?